vnext/Microsoft.ReactNative.ComponentTests/Microsoft.ReactNative.ComponentTests.vcxproj

```diff
@@ -158,6 +164,13 @@
     </ClCompile>
     <ClInclude Include="CommonReaderTest.h" />
     <ClInclude Include="pch/pch.h" />
+    <ClCompile Include="$(JSI_SourcePath)\jsi\test\testlib.cpp">
+      <!--
+        C4245 - 'initializing': conversion from 'int' to 'unsigned __int64', signed/unsigned mismatch
+        C4389 - '==': signed/unsigned mismatch
+      -->
+      <DisableSpecificWarnings>4245;4389;$(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
>4245;4389 [](start = 31, length = 9)

OOC, couldn't these warnings be quelled via static_casts in code? If so, what makes disabling them the better choice?

Reply:
The testlib.cpp is used from the RN npm. To fix it we need to change Hermes and RN code. This is why it is easier just to disable these warnings for now.

---
In reply to: [470189825](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470189825) [](ancestors = 470189825)

Reply:
can we just disable them at the point where they are produced instead of disabling them everywhere in the project?

---
In reply to: [470244479](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470244479) [](ancestors = 470244479,470189825)

Reply:
This file is coming from the NPM package. It is not part of our source tree. We cannot change the file. So, we are doing the next best thing: disable them for this file only in the project file. These warnings are not disabled for the whole project, they are disabled only for one file which comes from the RN NPM package. Note that they are inside of the ClCompile for testlib.cpp file only.

---
In reply to: [470473232](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470473232) [](ancestors = 470473232,470244479,470189825)

Reply:
gotcha, github was only showing me a snippet and i hadn't clicked the little arrows to show where this was in